apps/job-info-consumer/consumer/go.sum

```diff
@@ -65,9 +65,12 @@ github.com/Microsoft/go-winio v0.6.1 h1:9/kr64B9VUZrLm5YYwbGtUJnMgqWVOdUAXu6Migc
 github.com/Microsoft/go-winio v0.6.1/go.mod h1:LRdKpFKfdobln8UmuiYcKPot9D2v6svN5+sAH+4kjUM=
```

Comment:
bid add this file is change while its corresponding go.mod is unchanged. I suspect this might not be necessary.